frontend/sikkim-earthquake-2011-explorer/style.css

```
@@ -0,0 +1,351 @@
+:root {
+    --bg: #101820;
+    --bg-alt: #16202b;
+    --surface: #1c2836;
+    --line: #314155;
+    --text: #eef3f8;
+    --muted: #b4c2d1;
+    --dim: #8a9aab;
+    --accent: #c9a227;
+    --accent-2: #7eb8c9;
+    --heading: "Playfair Display", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)